### DISCUSSION TITLE
NHL Channels

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#182] [Q&A] [answered]
Hi,

I would like to please know, is the NHL channels included in the main database: https://iptv-org.github.io/iptv/index.m3u

If not, how would I go about requesting it please?

Thank you so much for your reply!

**Comment by what-a** [comment, marked as answer]
No, there isn't anything equivalent to channel packages such as "NHL Center Ice" nor "NHL Live."  There's only a few games on CBC (geo-blocked outside of Canada) and on TNT (geo-blocked outside of USA).

Here's the link to request a channel:
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel

**Reply by jphilippe2017** [reply]
Thanks so much for your answer, I will request it via your request a channel link.
